fili-core/src/main/java/com/yahoo/bard/webservice/web/responseprocessors/DruidJsonResponseContentKeys.java

```diff
@@ -11,7 +11,9 @@
     UNCOVERED_INTERVALS_OVERFLOWED("uncoveredIntervalsOverflowed"),
     STATUS_CODE("status-code"),
     RESPONSE("response"),
-    ETAG("Etag")
+    ETAG("Etag"),
+    REQUEST_ETAG("If-None-Match"),
+    NON_EXISTING_ETAG_VALUE("non-existing-etag")
```

Comment:
We should start a new `DruidRequestRequestContentKeys` and put the request stuff there.